src/App.js

```diff
@@ -1,12 +1,62 @@
-import React from 'react';
+//import React, {useEffect, useState} from 'react';
 import './App.css';
+import PokemonCard from "./PokemonCard/PokemonCard";
+import {useEffect, useState} from "react";
+import axios from "axios";
+import pokemonLogo from "./assets/Daco_5394286.png"
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [endPoint, setEndPoint] = useState('https://pokeapi.co/api/v2/pokemon/')
+    const [pokemons, setPokemons] = useState('')
```

Comment:
Als je een makkelijk onderscheid wil hebben tussen of het al wél of niet is ingeladen, zou ik hier voor default waarde van `undefined` gaan
```suggestion
    const [pokemons, setPokemons] = useState();
```